### PR TITLE
Create bug_bounty.md

### PR DESCRIPTION
**Pull request by rcai1**

### CODE REVIEW

File: documentation/synthetic_tokens/security/bug_bounty.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+The scope of our bug bounty program includes core smart contracts related to release [v1.0] of the DVM and financial templates. 
+-->
+It does not include known issues with the intended behavior. 
+<!-- TODO: create overview document for system
+Please refer to the [readme file] in the repository for an extensive overview of the intended behavior of the system.
+--> 
+
+### In scope:
+* DVM contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/oracle/implementation)
+* Multi-sponsor token contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/financial-templates/implementation)
```
Comment (hlambur) [inline documentation/synthetic_tokens/security/bug_bounty.md:35]: Technically we don't want this to be part of the bug bounty until after we get OZ's audit back. 

@mrice32 what do you think of making a note stating that we're accepting bugs for the DVM now, and that we'll be accepting bugs for the NFCT after say May 1?

File: documentation/synthetic_tokens/security/bug_bounty.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# UMA Bug Bounty Program
+
+Security of the platform is our highest priority. 
+All contract code and balances are publicly verifiable, and security researchers are eligible for a bug bounty for reporting undiscovered vulnerabilities.
+
+## Audit
+<!-- TODO: Add details of audit. 
+The contracts have been carefully audited by smart contract security experts [names]. 
+The audit report can be found [here].
+-->
```
Comment (allisonlu22) [inline documentation/synthetic_tokens/security/bug_bounty.md:10]:
```suggestion
--> test change
```

File: documentation/synthetic_tokens/security/bug_bounty.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+--> 
+
+UMA encourages the community to audit our contracts and security; we also encourage the responsible disclosure of any issues.
+The bug bounty program is intended to recognize the value of working with the community of independent security researchers, and sets out our definition of good faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+## Rewards
+UMA offers substantial rewards for discoveries that can prevent the loss of assets, the freezing of assets, or harm to users. 
+
+Rewards range from $500 to $25,000. 
+All rewards will be paid in ETH, and the amount of compensation will vary depending on Severity.
```
Comment (mrice32) [inline documentation/synthetic_tokens/security/bug_bounty.md:19]: Severity is capitalized in this sentence but not in the following sentence.
```suggestion
All rewards will be paid in ETH, and the amount of compensation will vary depending on severity. 
```

File: documentation/synthetic_tokens/security/bug_bounty.md (hunk 1 above)
Comment (mrice32) [reply documentation/synthetic_tokens/security/bug_bounty.md:33]: That seems reasonable. We could also just remove this line and add when we're ready to add it to the scope. Either way SGTM.

File: documentation/synthetic_tokens/security/bug_bounty.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+## Terms & Conditions
+If you comply with the policies below when reporting a security issue to us, we will not initiate a lawsuit or law enforcement investigation against you in response to your report.
+
+We ask that you:
+* Report any vulnerability you’ve discovered promptly.
+* Avoid violating the privacy of others, disrupting our systems, destroying data, or harming user experience.
+* Use only bugs@umaproject.org to discuss vulnerabilities with us.
+<!-- TODO: Clarify what "fixed" means 
+* Keep the details of any discovered vulnerabilities confidential until they are fixed and publicly announced by the UMA {foundation?, team?} on {Medium, docs.umaproject.org/bugs, twitter, somewhere else?}.
```
Comment (mrice32) [inline documentation/synthetic_tokens/security/bug_bounty.md:62]: Why not just remove fixed and just leave the announced part? Then we don't need to clarify what `fixed` means in this context. cc @allisonlu22

Comment (rcai1) [reply documentation/synthetic_tokens/security/bug_bounty.md:62]: That works for me. cc @hlambur

Comment (hlambur) [reply documentation/synthetic_tokens/security/bug_bounty.md:62]: agreed. do that

File: documentation/synthetic_tokens/security/bug_bounty.md (hunk 1 above)
Comment (rcai1) [reply documentation/synthetic_tokens/security/bug_bounty.md:33]: It makes sense to me to remove this line (comment it out) and add it back when we're ready.

File: documentation/synthetic_tokens/security/bug_bounty.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Rewards range from $500 to $25,000. 
+All rewards will be paid in ETH, and the amount of compensation will vary depending on severity. 
+The severity is calculated according to the [OWASP](https://owasp.org/www-project-risk-assessment-framework/) risk rating model based on Impact and Likelihood.
+
+![](severity.png)
+
+## Scope
+<!-- TODO: add version name of DVM 
+The scope of our bug bounty program includes core smart contracts related to release [v1.0] of the DVM and financial templates. 
+-->
```
Comment (rcai1) [inline documentation/synthetic_tokens/security/bug_bounty.md:27]: Are we agreed on using v1.0 as the version name for the DVM? @mrice32 @hlambur @allisonlu22

Comment (mrice32) [reply documentation/synthetic_tokens/security/bug_bounty.md:27]: SGTM

File: documentation/synthetic_tokens/security/bug_bounty.md (hunk 4 above)
Comment (rcai1) [reply documentation/synthetic_tokens/security/bug_bounty.md:64]: Great - so I'll leave this as "announced by the UMA Foundation on the Documentation website." Does that work?

File: documentation/synthetic_tokens/security/bug_bounty.md (hunk 5 above)
Comment (rcai1) [reply documentation/synthetic_tokens/security/bug_bounty.md:27]: Great, will uncomment this. Aside from this, I think this is ready to be merged, but will defer to @hlambur to merge.

**Pull request merged**